roottest/root/treeformula/schemaEvolution/schemaRun1.C

```diff
@@ -1,26 +1,12 @@
+int schemaRun1()
 {
-#if  defined(ClingWorkAroundIncorrectTearDownOrder)
```

Comment:
Those were intentionally left as marker of known issues to be resolved/investigated later.  If we are removing them from the test they should be replaced by github issue with a reproducer(s) (or if they are resolved already we should remove the workarounds everywhere).

Reply:
But it is very generic question - should one mix testing of very different issues in one macro?
`ClingWorkAroundIncorrectTearDownOrder` is about cling cleanup problem in unnamed macros.
But should we test/handle this problem together with schema evolution in TTreeFormula?
Why not simplify scripts and concentrate only on one concrete problem?

Reply:
`ClingWorkAroundIncorrectTearDownOrder` used in many other places.
And by using this `#define` we do not test some feature - but workaround problem.

Reply:
`ClingWorkAroundIncorrectTearDownOrder` still appears in 7 other tests.
This defines is not set - neither in cmake not in old `FixCling.mk` file.
It is associated with issue https://its.cern.ch/jira/browse/ROOT-4768, which was fixed before ROOT 5.28.
Is it really important to restore it?